Smarty/templates/Components/MasterDetail/GridHeader.tpl

```diff
@@ -26,6 +26,11 @@
 <div class="slds-page-header__controls">
 <div class="slds-page-header__control">
 <div class="slds-button-group" role="group">
+{if !empty($MasterDetailLayoutMap.toolbar.massedit)}
+<button type="button" class="slds-button slds-button_neutral" title="{$APP.LBL_MASS_EDIT}" onclick="masterdetailwork.MDMassEditRecords(this, '{$MasterDetailLayoutMap.targetmodule}', '{$MasterDetailLayoutMap.mapname}')" data-id="{$MasterDetailLayoutMap.mapname}--massedit" id="btn-{$MasterDetailLayoutMap.mapname}--massedit">
```

Comment:
can we change the `--` to underscore? `--` is a javascript operator and it is easy to conflict or misinterpret

Reply:
Sure, I let me update it.